packages/xo-server/src/api/vm.js

```diff
@@ -1566,7 +1566,7 @@ setBootOrder.params = {
 }
 
 setBootOrder.resolve = {
-  vm: ['vm', 'VM', 'operate'],
+  vm: ['vm', ['VM', 'VM-template'], 'operate'],
```

Comment:
Why this change?

Reply:
Required for tests see https://github.com/vatesfr/xen-orchestra/pull/5119/files#diff-c2fc74aff8ab2887a8326f05b1edce48R58

Reply:
I'm not sure it's a good idea to expose a way to mutate templates but let's go for it.